grails-app/views/purchaseOrder/_enterOrderDetails.gsp

```diff
@@ -2,10 +2,10 @@
 <%@ page import="org.pih.warehouse.order.OrderType" %>
 <%@ page import="org.pih.warehouse.order.OrderTypeCode" %>
 
+<g:applyLayout name="custom">
 <html>
 <head>
 <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
-<meta name="layout" content="custom" />
```

Comment:
This meta tag doesn't apply the layout on those two files. During searching for possible solutions/problems to solve, I found this [stackOverflow discussion](https://stackoverflow.com/questions/9629155/using-3-layouts-on-same-gsp-page) which is not too much connected to this issue, but I found there `g:applyLayout` and it works as it should.